recipes/coz/all/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
:thinking: Why write an entire build scrip if upstream already has one?

You can use the Autotools for make file projects... this would eliminate us having to maintain this cmake in the future.

Reply:
The upstream uses CMake to build now.